lnst/Recipes/ENRT/GreOvsTunnelRecipe.py

```diff
@@ -97,25 +101,24 @@ def create_tunnel(self, configuration):
         Integration ports are configured with IPv4 and IPv6 addresses
         of the tunneled networks.
         """
-        endpoint1, endpoint2 = configuration.tunnel_endpoints
+        endpoint1, endpoint2 = tunnel_endpoints
         m1 = endpoint1.netns
         m2 = endpoint2.netns
         ip_filter = {"family": AF_INET}
 
-        for i, (host, endpoint) in enumerate([(m1, endpoint2), (m2, endpoint1)]):
+        for i, (host, endpoint) in enumerate([(m1, endpoint2), (m2, endpoint1)], 1):
             remote_ip = endpoint.ips_filter(**ip_filter)[0]
             host.br0 = OvsBridgeDevice()
             host.int0 = host.br0.port_add(interface_options={"type": "internal"})
-            configuration.tunnel_devices.append(host.int0)
-            host.int0.ip_add(ipaddress("192.168.200." + str(i + 1) + "/24"))
-            host.int0.ip_add(ipaddress("fc00::" + str(i + 1) + "/64"))
+            config.configure_and_register_ip(host.int0, ipaddress(f"192.168.200.{i}/24"))
+            config.configure_and_register_ip(host.int0, ipaddress(f"fc00::{i}/64"))
 
             host.br0.tunnel_add("gre", {"options:remote_ip": remote_ip})
 
             host.br0.up()
             host.int0.up()
 
-        self.wait_tentative_ips(configuration.tunnel_devices)
```

Comment:
You removed this from the code, so we will now won't wait for tentative flag.

Reply:
Accidentally removed, I returned it back there.